src/it/ajc_noclasses/verify.groovy

```diff
@@ -34,4 +34,4 @@
 */
 
 assert new File(basedir, 'target/classes').list().length == 0
-assert new File(basedir, 'target/unwoven').list().length == 0
+assert new File(basedir, 'target/unwoven').list() == null
```

Comment:
@mkordas because the whole copy logc is inside ``if(hasClasses())`` (it was like this before, except the ``.mkdirs()`` call). So before, the unwoven folder was created empty if there weren't any classes to copy. Now it is created only if there are classes to copy. 

Above, ``new File(basedir, 'target/unwoven').list()`` is null because ``target/unwoven`` does not exist (the test is for no classes case)